srcpkgs/python-logbook/template

```diff
@@ -0,0 +1,31 @@
+# Template file for 'python-logbook'
+pkgname=python-logbook
```

Comment:
Please make it python 3 only, we do not want new python 2 packages.

Reply:
Good point.